bindings/python/setup.py

```
@@ -58,6 +59,35 @@
 if len(__version__.split("+")) > 1 :
     buildCommitHash = __version__.split("+")[1]
 
+repo_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+source_license = os.path.join(repo_root, "LICENSE")
+source_notices_dir = os.path.join(repo_root, "notices")
+wheel_license_output_dir = os.path.join(here, "generated", "wheel_licenses")
+packaged_license_dir = os.path.join(here, "generated", "license")
+packaged_license = os.path.join(packaged_license_dir, "LICENSE")
+packaged_notices_dir = os.path.join(here, "generated", "notices")
+
+def _stage_license_files():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_stage_license_files`

Add return type annotation: `None`

(ANN202)